helper_apps/MediaFileChunker/blobHandler.js

```diff
@@ -0,0 +1,151 @@
+import fs from 'fs';
+import path from 'path';
+import { BlobServiceClient } from '@azure/storage-blob';
+import { v4 as uuidv4 } from 'uuid';
+import Busboy from 'busboy';
+import { PassThrough } from 'stream';
+import { pipeline as _pipeline } from 'stream';
+import { promisify } from 'util';
+const pipeline = promisify(_pipeline);
+import { join } from 'path';
+
+
+import { publicFolder, port, ipAddress } from "./start.js";
+
+const getBlobClient = () => {
+    const connectionString = process.env.AZURE_STORAGE_CONNECTION_STRING;
+    const containerName = process.env.AZURE_STORAGE_CONTAINER_NAME;
+    if (!connectionString || !containerName) {
+        throw new Error('Missing Azure Storage connection string or container name environment variable');
+    }
+
+    const blobServiceClient = BlobServiceClient.fromConnectionString(process.env.AZURE_STORAGE_CONNECTION_STRING);
+    const containerClient = blobServiceClient.getContainerClient(process.env.AZURE_STORAGE_CONTAINER_NAME);
+
+    return { blobServiceClient, containerClient };
+}
+
+async function saveFileToBlob(chunkPath, requestId) {
+    const { containerClient } = getBlobClient();
+    // Use the filename with a UUID as the blob name
+    const blobName = `${requestId}/${uuidv4()}_${path.basename(chunkPath)}`;
+
+    // Create a read stream for the chunk file
+    const fileStream = fs.createReadStream(chunkPath);
+
+    // Upload the chunk to Azure Blob Storage using the stream
+    const blockBlobClient = containerClient.getBlockBlobClient(blobName);
+    await blockBlobClient.uploadStream(fileStream);
+
+    // return blobName;
```

Comment:
Is this just for debug?

Reply:
this is the actual azure file blob saver. or you mean the commented out lines?

Reply:
Just the commented out line.